frontend/scss/components/organisms/news.scss

```diff
@@ -77,13 +68,21 @@
       padding-right: 0;
     }
 
+    div {
```

Comment:
This would add the `transition` property to all `ap-o-news-card-title`, `ap-o-news-card-image` and `ap-o-news-card`. Can we use a more specific selector here?